### PR TITLE
[SMALLFIX] Add javadoc for MessageSavingHandler#getMessage, MessageSavingHandler#reset

### PR DESCRIPTION
**Pull request by shizihao123**
[SMALLFIX] Add javadoc for MessageSavingHandler#getMessage, MessageSavingHandler#reset

### CODE REVIEW

File: tests/src/test/java/alluxio/network/protocol/MessageSavingHandler.java [hunk 1]
```diff
@@ -37,7 +37,11 @@ public void channelRead0(ChannelHandlerContext ctx, RPCMessage message) {
     mMessageAvailable.release();
   }
 
-  // Returns the received message. This is blocking.
+  /**
+   * This is blocking.
```
Comment (gpang) [inline tests/src/test/java/alluxio/network/protocol/MessageSavingHandler.java]: This should be "Returns the received message. This is blocking."

Comment (shizihao123) [inline tests/src/test/java/alluxio/network/protocol/MessageSavingHandler.java]: I agree with you and  have fixed that, thx .

**Comment by shizihao123** [comment]
I have fixed that , thx.

**Comment by gpang** [comment]
@shizihao123 Thanks for these improvements!

**Pull request merged**